tests/e2e/pv_services/test_del_mon_service_and_create_pvc.py

```diff
@@ -0,0 +1,192 @@
+import json
```

Comment:
Can we place this file under "tests/manage/pv_services/"?

Reply:
moved to tests/manage/pv_services/